server/Service/Services/EmailSender.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using MailKit.Net.Smtp;
+using MimeKit;
+using Service.Interfaces;
+using Service.Models;
+
+namespace Service.MailService
+{
+  public class EmailSender : IEmailSender
+  {
+    private readonly EmailConfiguration _emailConfig;
+
+    public EmailSender(EmailConfiguration emailConfig)
+    {
+      _emailConfig = emailConfig;
+    }
+
+    public void SendEmail(Message message)
+    {
+      var emailMessage = CreateEmailMessage(message);
+
+      Send(emailMessage);
+    }
+
+    public async Task SendEmailAsync(Message message)
+    {
+      var mailMessage = CreateEmailMessage(message);
+
+      await SendAsync(mailMessage);
+    }
+
+    private MimeMessage CreateEmailMessage(Message message)
+    {
+      var emailMessage = new MimeMessage();
+      emailMessage.From.Add(new MailboxAddress(_emailConfig.From));
+      emailMessage.To.AddRange(message.To);
+      emailMessage.Subject = message.Subject;
+
+      var bodyBuilder = new BodyBuilder { HtmlBody = string.Format("<h2 style='color:red;'>{0}</h2>", message.Content) };
+
+      if(message.Attachments != null && message.Attachments.Any())
```

Comment:
do you really have any attachments in your email? do you add images or files to it or just plain text or html text?

Reply:
not yet. this may be necessary in the future